.github/actions/add-azure-application-insights-release-annotation/action.yml

```diff
@@ -20,7 +20,7 @@ runs:
     - name: Setup Scripts
       shell: pwsh
       run: |
-        "${{ github.action_path }}" >> $Env:GITHUB_PATH
+        '${{ github.action_path }}' >> $Env:GITHUB_PATH
```

Comment:
I changed this pattern everywhere (using single quotes prevents accidental PowerShell variable injection), hence all the @reference changes too.

Reply:
lol RIP everyone with the cute programming account names who get `@`-ed all the time